### ISSUE TITLE
"Unpermitted parameter: force" when updating stock count on hand

### ISSUE THREAD

**Issue by travisp** [solidusio/solidus#652]
```
Started PUT "/api/stock_locations/1/stock_items/1" for ::1 at 2016-01-05 10:10:54 -0500
Processing by Spree::Api::StockItemsController#update as JSON
  Parameters: {"stock_item"=>{"backorderable"=>"false", "count_on_hand"=>"5", "force"=>"true"}, "stock_location_id"=>"1", "id"=>"1"}
```
```
Unpermitted parameter: force
```

It appears that ```Spree::Api::StockItemsController#update``` uses the "force" param directly, but then does not remove it from the params like the code does for count_on_hand before passing through to update_attributes.

While there aren't direct consequences of this, it prevents updating stock from working with the default development and test configuration which raises when passing ignored attributes (to better catch mistakes). Happy to add a small PR to fix.

**Comment by cbrunsdon** [comment]
@travisp nice catch, thanks. Aye, if its not too much trouble a PR would be appreciated.

Here's the code he's talking about for posterity:
https://github.com/solidusio/solidus/blob/master/api/app/controllers/spree/api/stock_items_controller.rb#L36-L39

```ruby
def update
  @stock_item = StockItem.accessible_by(current_ability, :update).find(params[:id])
  @stock_location = @stock_item.stock_location

  adjustment = count_on_hand_adjustment
  params[:stock_item].delete(:count_on_hand)
  adjustment -= @stock_item.count_on_hand if params[:stock_item][:force]

  Spree::StockItem.transaction do
    if @stock_item.update_attributes(stock_item_params)
      adjust_stock_item_count_on_hand(adjustment)
      respond_with(@stock_item, status: 200, default_template: :show)
    else  
      invalid_resource!(@stock_item)
    end   
  end
end
```

**Comment by ajkamel** [comment]
Im seeing the same issue. I'll see if i can work on the PR.

**Comment by cbrunsdon** [comment]
This is fixed in #811, thanks @travisp for reporting and @ajkamel for making the PR.
